neurokit2/hrv/hrv_frequency.py

```diff
@@ -53,8 +53,8 @@ def hrv_frequency(
         If False, warnings will be printed. Default to True.
     show : bool
         If True, will plot the power in the different frequency bands.
-    norm : bool
-        Normalization of power.
+    normalization : bool
+        Normalization of power. Default to True.
```

Comment:
@Tam-Pham could you add a tinybitsy more details here like why would someone want to normalize or not